09 Redux/src/actions/loadMember.ts

```diff
@@ -0,0 +1,8 @@
+const loadMember = (id : number) => {
+   return {
+     type: 'MEMBER_LOAD'
+     ,id: id
```

Comment:
We have to find an standard way, I like to place it at the beginning, because when I remove a line sometimes I get a mess (error).